presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestPrestoDriverUri.java

```diff
@@ -268,6 +269,16 @@ public void testUriWithCustomHeaders()
         assertEquals(properties.getProperty(CUSTOM_HEADERS.getKey()), customHeaders);
     }
 
+    @Test
```

Comment:
Please add a test for failed validation?

Reply:
@tdcmeehan Sure, working on that.